### PR TITLE
Remote Tickets info

### PR DESCRIPTION
**Pull request by laisbsc**
Add page with remote ticket information.

### CODE REVIEW

File: src/content/pages/remote.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+subtitle: Join EuroPython 2024 remotely!
+social_card: social_card_remote.png
+---
+
+# EuroPython 2024 Remote
+
+As happy as we are about finally being able to see many friends in **Prague**
+this year, we are also aware that many of you will not be able to make it in-person.
+Again, we present you EuroPython 2024 remote:
+wherever you are based, let's celebrate Python and our community, rejoice at our diversity and
```
Comment (cybit) [inline src/content/pages/remote.mdx:12]:
```suggestion
Wherever you are based, let's celebrate Python and our community, rejoice at our diversity,
```

File: src/content/pages/remote.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+learn and have fun in Prague and beyond! In this spirit, we strive to make the remote participation
+as interactive as possible.
+
+### How to participate in the conference remotely?
+
+- To participate in the conference interactively (with guaranteed access to livestreams), a "remote ticket" registration is required. See details and pricing on the [Tickets information](/tickets#remote-tickets) page;
```
Comment (cybit) [inline src/content/pages/remote.mdx:18]:
```suggestion
- To participate in the conference interactively (andguaranteed access to livestreams), a "remote ticket" registration is required. See details and pricing on the [Tickets information](/tickets#remote-tickets) page.
```

File: src/content/pages/remote.mdx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- All ticket holders will receive a link to join our Discord server. We will send out invites for you to join our server closer to the conference.
```
Comment (cybit) [inline src/content/pages/remote.mdx:19]:
```suggestion
- All ticket holders will receive a link to join our Discord server for the conference.
```

File: src/content/pages/remote.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+*Note: in-person ticket holders will be provided with remote event access.*
```
Comment (cybit) [inline src/content/pages/remote.mdx:21]:
```suggestion
*Note: in-person ticket holders will be provided with remote event access as well.*
```

File: src/content/pages/remote.mdx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+### What is included in the remote ticket?
+Remote participation is between 10-12 July (Wednesday - Friday):
```
Comment (cybit) [inline src/content/pages/remote.mdx:24]:
```suggestion
Remote participation between 10-12 July (Wednesday - Friday):
```

File: src/content/pages/remote.mdx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+- Livestreams of Keynotes, Panel discussions, all 6 tracks of talks.
```
Comment (cybit) [inline src/content/pages/remote.mdx:25]:
```suggestion
- Livestreams of Keynotes, Panel discussions, All 6 tracks of talks.
```
or change Panel to panel.

File: src/content/pages/remote.mdx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+- Live participation (text-based) on Q&A session post-talks.
+- Text-based discussions with available speakers.
+- Online job board and chat with sponsors at the remote booths.
+- On-Demand viewing, which will be available hours after the live stream.
+
+*Note: Tutorials and Workshops on 8-9 July will not be streamed.*
```
Comment (cybit) [inline src/content/pages/remote.mdx:31]:
```suggestion
*Note: Tutorials and Workshops on 8 and 9 July will not be streamed.*
```

File: src/content/pages/remote.mdx (hunk 2 above)
Comment (egeakman) [reply src/content/pages/remote.mdx:18]: Oops

File: src/content/pages/remote.mdx (hunk 6 above)
Comment (egeakman) [reply src/content/pages/remote.mdx:25]: Panel to panel makes more sense IMHO

File: src/content/pages/remote.mdx [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+### What changes ###
+This year, we will **NOT** provide open and free access to the livestreams of our conference talks (19-21 July). If you would like to watch EuroPython remotely, please consider purchasing a remote ticket.
```
Comment (hypha) [inline src/content/pages/remote.mdx:34]:
```suggestion
This year, we will **NOT** provide open and free access to the livestreams of our conference talks (10-12 July). If you would like to watch EuroPython remotely, please consider purchasing a remote ticket.
```

File: src/content/pages/remote.mdx (hunk 2 above)
Comment (cybit) [reply src/content/pages/remote.mdx:18]: The oops was the missing space?

File: src/content/pages/remote.mdx [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+- Livestreams of Keynotes, Panel discussions, all 6 tracks of talks.
+- Live participation (text-based) on Q&A session post-talks.
+- Text-based discussions with available speakers.
+- Online job board and chat with sponsors at the remote booths.
+- On-Demand viewing, which will be available hours after the live stream.
+
+*Note: Tutorials and Workshops on 8 and 9 July will not be streamed.*
+
+### What changes ###
+This year, we will **NOT** provide open and free access to the livestreams of our conference talks (10-12 July). If you would like to watch EuroPython remotely, please consider purchasing a remote ticket.
```
Comment (anivorlis) [inline src/content/pages/remote.mdx:34]: ... purchasing a remote ticket. Purchasing a remote ticket not only gives you access to the conference but also helps support the community!

File: src/content/pages/remote.mdx (hunk 2 above)
Comment (egeakman) [reply src/content/pages/remote.mdx:18]: > The oops was the missing space?

Yes! It's all good now.

File: src/content/pages/remote.mdx (hunk 6 above)
Comment (cybit) [reply src/content/pages/remote.mdx:25]: updated suggestion.

File: src/content/pages/remote.mdx (hunk 9 above)
Comment (egeakman) [reply src/content/pages/remote.mdx:34]:
```suggestion
This year, we will **NOT** provide open and free access to the livestreams of our conference talks (10-12 July).
If you would like to watch EuroPython remotely, please consider purchasing a remote ticket.
Purchasing a remote ticket not only gives you access to the conference but also helps support the community!
```

Comment (egeakman) [reply src/content/pages/remote.mdx:34]: @laisbsc Is this suggestion OK to commit?

**Review by egeakman — APPROVED** [review]
Thanks Laís!

**Comment by cybit** [comment]
All resolved. Let's merge.

**Pull request merged**